tsl/src/compression/compression_dml.c

```diff
@@ -1710,7 +1710,7 @@ key_column_is_null(tuple_filtering_constraints *constraints, Relation chunk_rel,
 		const NameData *attname = attnumAttName(chunk_rel, chunk_attno);
 
 		AttrNumber ht_attno = get_attnum(ht_relid, NameStr(*attname));
-		if (slot_attisnull(slot, ht_attno))
+		if (slot_attisnull(slot, ht_attno) && !get_attgenerated(ht_relid, ht_attno))
```

Comment:
I don't understand this condition, can you add a comment?

Reply:
Yeah, this is just to exclude NULL value check for generated columns.

I'll add a comment.